src/components/ListDetail/ListDetail.module.css

```diff
@@ -111,12 +114,14 @@
 
 /**
  * List detail link
+ * 1) Base style for each link in the list
+ * 2) Using disabled text color here because all links are "inactive" until selected and this color is overridden
  */
 .list-detail__link {
   position: relative;
   display: flex;
   text-decoration: none;
-  color: var(--eds-theme-color-neutral-subtle-foreground);
+  color: var(--eds-theme-color-text-disabled); /* 2 */
```

Comment:
Flagging that this is reallyy hard to read color contrast wise. We may want to pick a darker disabled text color

Reply:
Good point.